src/App.js

```diff
@@ -22,6 +24,7 @@ class App extends Component {
             Tambolify!
           </button>
           <LastValue value={this.state.nextSelectedValue} />
+          <LastFiveValues value={this.state.lastFiveSelectedValues} />
```

Comment:
I don't know if we actually want to do this, but it should be quite easy to make the number of previous values shown configurable (not in the UI, but at least easily changeable in code). 

Even if we decide not to do this now, renaming this (and other variables in this file) to not have the word 'five' in them might be a good idea.